controller/src/beerocks/master/db/db.h

```diff
@@ -394,6 +394,42 @@ class db {
     bool set_hostap_is_acs_enabled(std::string mac, bool enable);
     bool get_hostap_is_acs_enabled(std::string mac);
 
+    //
+    // Channel Scan
+    //
+    bool set_channel_scan_is_enabled(const std::string &mac, const bool enable);
+    bool get_channel_scan_is_enabled(const std::string &mac);
```

Comment:
So it will be
> channel_scan_enabled(const std::string &mac) const; //get
> channel_scan_enabled(const std::string &mac, const bool enable); //set

?

Reply:
There are far more important things to fix in this PR. So let's keep it for #665